indico/modules/rb/client/js/common/map/reducers.js

```diff
@@ -25,7 +25,13 @@ export default combineReducers({
     areas: (state = [], action) => {
         switch (action.type) {
             case mapActions.AREAS_RECEIVED:
+            case mapActions.MAP_AREA_UPDATED:
+            case mapActions.MAP_AREA_CREATED:
```

Comment:
I'd keep admin stuff separate and just reload the list of map areas when leaving the admin area

Reply:
But I need this to rerender the map with new areas

Reply:
hadn't noticed that you are using the same state for admin and the normal rb interface until after writing that comment

Reply:
I mean, is there any advantage of putting the same data in the admin-related part of redux store?

Reply:
not if the data is the same which seems to be the case here

Reply:
Yep, exactly.